### DISCUSSION TITLE
litchi_wp integration

### DISCUSSION THREAD

**Discussion by JoeKae** [YarostheLaunchpadder/MissionPlanner-to-Litchi#15] [Ideas]
# Idea
I added support for parsing raw Litchi waypoint csv files in [litchi_wp](https://pypi.org/project/litchi-wp/).
I think it is a good idea to publish a pypi package for Mission Planner waypoints. Then both packages could have a method to import a list of the other module. See the example to make it more clear.
Thus the possibilities of the integration into other applications would be versatile.

So i suggest we create a Github Org to Bundle all the tools. Name suggestions welcome.

## Example:
```python
mp_waypoints = mp_wp.Waypoint.from_file(mp_filename)
litchi_waypoints = litchi_wp.Waypoint.from_file(litchi_filename)

litchi_waypoints_2 = litchi_wp.Waypoints.from_mp_waypoints(mp_waypoints)
mp_waypoints_2 = mp_wp.Waypoint.from_litchi_waypoints(litchi_waypoints)
"""
Now mp_waypoints and mp_waypoints_2 should be identical
and litchi_waypoints and litchi_waypoints_2 should be identical
"""
```

**Comment by YarosMallorca** [comment]
Let's create a GitHub Organization and move all tools to there. It's a good idea!
What do you think is a good name for it?

**Reply by JoeKae** [reply]
- litchi-tools
- mission-planner-and-litchi-tools
- waypoint-conversion-tools
- flightplan-conversion-tools

You like any of those?
The  waypoint-conversion-tools and flightplan-conversion-tools leave the door open to include packages for other applications as well. Like QGIS and so on.
